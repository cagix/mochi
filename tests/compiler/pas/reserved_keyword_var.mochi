let record = 1
print(record)
