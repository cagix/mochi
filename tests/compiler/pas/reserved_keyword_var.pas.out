program main;
{$mode objfpc}
uses SysUtils;

type TIntArray = array of integer;

var
	_record: integer;

begin
	_record := 1;
	writeln(_record);
end.
